gitbook-docs/security.md

```diff
@@ -112,6 +126,92 @@ Clients using other kinds of protocols can include the `token` in the Signal K m
 }
 ```
 
+### Token Validation
+
+Tokens may have a short expiry time and need to be renewed periodically, or a token's current validity may be unknown. 
+
+#### Web Clients
+To validate a token, a web based client should send an HTTP GET request to `/signalk/«version»/auth/validate` with the token in the cookie.
```

Comment:
We should allow for token in header here as well, not just cookie.

Reply:
changed